### PR TITLE
chore: update for apple silicon

### PR DESCRIPTION
**Pull request by monfera**
## Summary

Updated dependencies for Apple silicon M1

### Checklist

- [ ] The proper **chart type** label has been added (e.g. `:xy`, `:partition`)
- [ ] The proper **feature** labels have been added (e.g. `:interactions`, `:axis`)
- [ ] The `:theme` label has been added and the `@elastic/eui-design` team has been pinged when there are `Theme` API changes
- [ ] All related issues have been linked (i.e. `closes #123`, `fixes #123`)
- [ ] New public API exports have been added to `packages/charts/src/index.ts`
- [ ] Unit tests have been added or updated to match the most common scenarios
- [ ] The proper documentation and/or storybook story has been added or updated
- [ ] The code has been checked for cross-browser compatibility (Chrome, Firefox, Safari, Edge)
- [ ] Visual changes have been tested with all available themes including `dark`, `light`, `eui-dark` & `eui-light`

### CODE REVIEW

File: packages/charts/src/reset_dark.scss [hunk 1]
```diff
@@ -1,6 +1,5 @@
-@import '~@elastic/eui/src/themes/eui/eui_colors_dark';
-@import 'style/themes/colors_dark';
```
Comment (markov00) [inline packages/charts/src/reset_dark.scss:2]: This actually doesn't exist in our code so I removed it

**Comment by monfera** [comment]
Thanks @markov00 it works locally with the pushed commit too, while CI VRT complains of `sass` related issues. As `master` works on M1 after addressing a local setup issue (literally, just the execution of `brew install pkg-config cairo pango libpng jpeg giflib librsvg` as on the `node-canvas` [site](https://github.com/Automattic/node-canvas/wiki/Installation%3A-Mac-OS-X)), it's your call if the timely `sass` changes should be carried through

**Comment by markov00** [comment]
@monfera let's close this PR for now if in master it works. We can switch the sass implementation later when we the migration to playwright is finished

**Comment by monfera** [comment]
Closing to defer the SASS changes, as with the right setup `master` already works on Apple Silicon

**Pull request closed without merging**